e2e/cypress/support/e2e.js

```diff
@@ -7,9 +7,19 @@ Cypress.Screenshot.defaults({
 
 const COMMAND_DELAY = 1000;
 
+function delay(ms) {
+  const end = Date.now() + ms;
+  while (Date.now() < end) {}
```

Comment:
I quite don't like this. IIRC there were some ideas about ignoring empty code blocks as a perf improvement in JS engines. Maybe one of the following two?

```js
function delay(ms) {
  let now = Date.now();
  const end = now + ms;
  

  do {
    now = Date.now();
  } while (now < end);
}
```

```js
async function delay(ms) {
  return new Promise((resolve) => setTimeout(resolve, ms));
}
```

However, I don't remember if Cypress supports:

```js
afterEach(async () => {
  await sleep(200);
});
```